mitiq/tests/test_folding_cirq.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import random
+import cirq
+from mitiq.folding_mitiq import local_folding, unitary_folding
+
+def random_circuit(depth):
+    """Returns a single-qubit random circuit based on Pauli gates."""
+    # defines qubit
+    q = cirq.GridQubit(0, 0)
+    circuit = cirq.Circuit()
+    for _ in range(depth):
+        circuit += random.choice([cirq.X(q), cirq.Y(q), cirq.Z(q)])
+    return circuit
+
+strech_vals = [1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.5, 4.5, 5.0]
+depth = 100
+
+def test_unitary_folding():
+    print("Testing unitary folding...")
```

Comment:
```suggestion
```
Can remove dangling print statements

Reply:
This applies to the others in this testing file. Using `pytest --pdb` one can jump into place with the debugger if there's any issues to check rather than needing the prints. This keeps the code simpler. If that isn't clear I'm happy to demo @rmlarose @andreamari :)